subprojects/docs/src/docs/userguide/extending-gradle/custom_gradle_types.adoc

```diff
@@ -207,17 +210,17 @@ Here is an example that shows a task type that receives a two services via prope
 [source.multi-language-sample,java]
 .Download.java
 ----
-include::{snippetsPath}/plugins/propertyServiceInjection/groovy/buildSrc/src/main/java/Download.java[]
+include::{snippetsPath}/plugins/propertyServiceInjection/groovy/buildSrc/src/main/java/Download.java[tags=download]
 ----
 ====
 
-[[nested_objects]]
-== Creating nested objects
+[[object_factory]]
```

Comment:
I would advise against changing the link id as it will potentially break existing links.